### PR TITLE
Fix typo in assertion log message

### PR DESCRIPTION
**Pull request by cbracken**
Fixes a minor typo in a debug log message in
EmbedderTestContextGL::SetupCompositor.

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I signed the [CLA].
- [X] All existing and new tests are passing.
- [ ] The reviewer has submitted any presubmit flakes in this PR using the [engine presubmit flakes form] before re-triggering the failure.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[engine presubmit flakes form]: https://forms.gle/Wc1VyFRYJjQTH6w5A

### CODE REVIEW

File: shell/platform/embedder/tests/embedder_test_context_gl.cc [hunk 1]
```diff
@@ -110,7 +110,7 @@ uint32_t EmbedderTestContextGL::GetWindowFBOId() const {
 }
 
 void EmbedderTestContextGL::SetupCompositor() {
-  FML_CHECK(!compositor_) << "Already ssetup a compositor in this context.";
+  FML_CHECK(!compositor_) << "Already setup a compositor in this context.";
   FML_CHECK(gl_surface_)
```
Comment (stuartmorgan) [inline shell/platform/embedder/tests/embedder_test_context_gl.cc:114]: This is actually still wrong; the verb is "set up".

**Review by stuartmorgan — APPROVED** [review]
LGTM with nit

File: shell/platform/embedder/tests/embedder_test_context_gl.cc (hunk 1 above)
Comment (cbracken) [reply shell/platform/embedder/tests/embedder_test_context_gl.cc:114]: Gah you’re right and I swear I’ve fixed that elsewhere to add to my shame :)

**Comment by cbracken** [comment]
Alright, I have set up 55 files worth of setup fixes for your review. Nouns and compound nouns remain setup. Also migrated the three instances of "tear-down" to "teardown" since that was more common in the code and also dominates both US and Commonwealth English search results.

**Comment by stuartmorgan** [comment]
> Alright, I have set up 55 files worth of setup fixes for your review.

Touché.

File: lib/web_ui/lib/src/engine/dom_renderer.dart [hunk 2]
```diff
@@ -464,7 +464,7 @@ flt-glass-pane * {
       // VisualViewport API is not enabled in Firefox as well. On the other hand
       // Firefox returns correct values for innerHeight, innerWidth.
       // Firefox also triggers html.window.onResize therefore we don't need this
-      // timer setup for Firefox.
+      // timer to be set up for Firefox.
```
Comment (stuartmorgan) [inline lib/web_ui/lib/src/engine/dom_renderer.dart:467]: I'd argue this was correct before, but I'll take it 🙂

File: shell/platform/android/io/flutter/plugin/common/EventChannel.java [hunk 3]
```diff
@@ -88,7 +88,7 @@ public void setStreamHandler(final StreamHandler handler) {
   }
 
   /**
-   * Handler of stream setup and tear-down requests.
+   * Handler of stream setup and teardown requests.
```
Comment (stuartmorgan) [inline shell/platform/android/io/flutter/plugin/common/EventChannel.java:91]: Should probably be "set up", but I can see an agreement for either being correct.

File: shell/platform/darwin/ios/framework/Source/FlutterEngine.mm [hunk 4]
```diff
@@ -824,7 +824,7 @@ - (FlutterBinaryMessengerConnection)setMessageHandlerOnChannel:(NSString*)channe
     return _connections->AquireConnection(channel.UTF8String);
   } else {
     NSAssert(!handler, @"Setting a message handler before the FlutterEngine has been run.");
-    // Setting a handler to nil for a not setup channel is a noop.
+    // Setting a handler to nil for a not yet setup channel is a noop.
```
Comment (stuartmorgan) [inline shell/platform/darwin/ios/framework/Source/FlutterEngine.mm:827]: This one definitely needs to be set up. I think in fact not-yet-set-up.

**Review by stuartmorgan — APPROVED** [review]
LGTM with one more.

File: lib/web_ui/lib/src/engine/dom_renderer.dart (hunk 2 above)
Comment (cbracken) [reply lib/web_ui/lib/src/engine/dom_renderer.dart:467]: Haha. I sat there for about 30 seconds debating whether to leave it ambiguous as to whether it was a "timer setup" or needing to have the timer set up before giving up and deciding to eliminate the ambiguity with a "to be".

File: shell/platform/darwin/ios/framework/Source/FlutterEngine.mm (hunk 4 above)
Comment (cbracken) [reply shell/platform/darwin/ios/framework/Source/FlutterEngine.mm:827]: Oh god I fixed the yet but apparently didn't save after hitting space.

Agreed. It should be not-yet-set-up though we should probably just rephrase. I realise that I, in particular, have a really bad habit of injecting incredibly-long-yet-dubious chains of words into my writing.

Comment (cbracken) [reply shell/platform/darwin/ios/framework/Source/FlutterEngine.mm]: Done.

**Pull request merged**